78812: Notifications - Alpha Docs (#1210)
replicatedhq/replicated-docs: docs/partials/instance-insights/_notifications-about.mdx

> Notifications can help catch problems before they happen and let you proactively contact customers to prevent support cases. For example, you can be notified of a degraded status and you can contact your customer about fixing it before the instance goes down. This approach can make issues quicker and easier to solve, and improve the customer experience with less down time.

Reviewer: Not sure if I did the link right, but good to link to "what this means" here

```suggestion
Notifications can help catch problems before they happen and let you proactively contact customers to prevent support cases. For example, you can be notified of a [degraded status](admin-console-display-app-status#resource-statuses) and you can contact your customer about fixing it before the instance goes down. This approach can make issues quicker and easier to solve, and improve the customer experience with less down time.
```

Notifications can help catch problems before they happen and let you proactively contact customers to prevent support cases. For example, you can be notified of a degraded status and you can contact your customer about fixing it before the instance goes down. This approach can make issues quicker and easier to solve, and improve the customer experience with less down time.

> For mor information about degraded statuses, see [Resource Statuses](admin-console-display-app-status#resource-statuses) in _Displaying Application Status_.

Reviewer: ```suggestion
For more information about degraded statuses, see [Resource Statuses](admin-console-display-app-status#resource-statuses) in _Displaying Application Status_.
```

> For mor information about degraded statuses, see [Resource Statuses](admin-console-display-app-status#resource-statuses) in _Displaying Application Status_.

Reviewer: ```suggestion
For more information about how application status is determined, see [Resource Statuses](admin-console-display-app-status#resource-statuses) in _Displaying Application Status_. For all instance events that may trigger notifications, see [Types of Events](instance-insights-event-data#types-of-events).
```
